x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/alerts_cell_actions.cy.ts

```diff
@@ -31,24 +31,23 @@ import {
   fillAddFilterForm,
   fillKqlQueryBar,
   openAddFilterPopover,
+  removeKqlFilter,
 } from '../../../tasks/search_bar';
 import { openActiveTimeline } from '../../../tasks/timeline';
 
 import { ALERTS_URL } from '../../../urls/navigation';
 
-describe.skip('Alerts cell actions', { tags: ['@ess', '@serverless'] }, () => {
-  before(() => {
+describe('Alerts cell actions', { tags: ['@ess', '@serverless'] }, () => {
+  beforeEach(() => {
```

Comment:
let's add `deleteAlertsAndRules();` from [here](https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/cypress/tasks/api_calls/common.ts#L38) to make sure each test will start with a clean state. While this is not really useful in ideal scenarios, if a test were to fail and were to be retried, we would have more than more alert in the table, which could cause more problems

Reply:
ok